ci/jenkins/pipelines/skuba-e2e-test.Jenkinsfile

```diff
@@ -31,7 +35,9 @@ node('caasp-team-private-integration') {
 
         if (env.REPO_BRANCH != ""){
                branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/5:/Branches:/${env.REPO_BRANCH}/SLE_15_SP2"
-        }
+               branch_registry = "registry.suse.de/devel/caasp/5/branches/${env.REPO_BRANCH}/containers/cr/containers/caasp/v5"
```

Comment:
it seems to me that you would need to check the `env.BRANCH_REGISTRY` or something like that instead of the `env.REPO_BRANCH` for all of these? or is that just me misunderstanding how this Jenkinsfile works?

Reply:
This value points to the IBS project by a convention we decided. So if you build a `Devel:CaaSP:5:Branches:cilium17` project `REPO_BRANCH` should be set to `cilium17`.

Then while you are right in the sense that this is not necessarily the same project for images and packages, I don't see any specific need to have them separate in the branch project. So lets assume that the images and packages for that branch are all placed in the same IBS project. This, as a user, allows you just to set a single value to make sure the CI is pulling packages and images from that specific IBS project.